src/Figure.js

```diff
@@ -2664,7 +2664,8 @@ draw2d.Figure = Class.extend(
         selectable: this.selectable,
         draggable: this.draggable,
         angle: this.rotationAngle,
-        userData: extend(true, {}, this.userData)
+        userData: extend(true, {}, this.userData),
+        isHub: this.IS_HUB || false
```

Comment:
sorry - no way. Adding an attribute in the base class just to identify if it is "any" sub class is not a good design decision